src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -492,7 +492,7 @@ private function getResourceObjectTypeFields(?string $resourceClass, ResourceMet
 
         if (null !== $resourceClass) {
             foreach ($this->propertyNameCollectionFactory->create($resourceClass) as $property) {
-                $propertyMetadata = $this->propertyMetadataFactory->create($resourceClass, $property, ['graphql_operation_name' => $mutationName ?? 'query']);
+                $propertyMetadata = $this->propertyMetadataFactory->create($resourceClass, $property, ['graphql_mutation_name' => $mutationName ?? 'query']);
```

Comment:
It seems strange to use `graphql_mutation_name` when it's a query. And there will be custom queries too.